src/App.jsx

```diff
@@ -14,33 +15,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [isReversed, setReversed] = useState(false);
+  const [sortType, setSortType] = useState('');
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortAlphabetically = () => {
+    setGoods(
+      [...goods].sort(
+        isReversed
+          ? (a, b) => b.localeCompare(a)
+          : (a, b) => a.localeCompare(b),
+      ),
+    );
+    setSortType('alphabeticall');
+  };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  const sortByLength = () => {
```

Comment:
see the previous comment about the sort function + you doubling a logic (reverse method)
try to create a function and reverse array in one place

Reply:
> see the previous comment about the sort function + you doubling a logic (reverse method) try to create a function and reverse array in one place

Good point